samcli/local/docker/lambda_container.py

```diff
@@ -50,8 +50,9 @@ def __init__(
         env_vars=None,
         debug_options=None,
         container_host=None,
-        container_host_interface=None,
+        container_host_interface=DEFAULT_CONTAINER_HOST_INTERFACE,
         extra_hosts=None,
+        container_host_interface=DEFAULT_CONTAINER_HOST_INTERFACE,
```

Comment:
Looks like this was duplicated accidentally. Additionally, looks like there was another update to a file in the develop branch causing issues with the container file here. I've started the tests anyways (hoping that they all don't get stuck at the duplicate argument). While we wait for the tests and the other conflict to be resolved, I'll raise this to the team to get another reviewer.

Aside from the PR jobs complaining, I don't have any concerns with your logic in the PR!

Reply:
ok, I deleted the extra line.